lms/djangoapps/oauth_dispatch/tests/constants.py

```diff
@@ -2,4 +2,4 @@
 Constants for testing purposes
 """
 
-DUMMY_REDIRECT_URL = u'https://example.edx/redirect'
+DUMMY_REDIRECT_URL = u'https://example.com/edx/redirect'
```

Comment:
example.com specifically exists to use for example code, and explicitly grants permission to use it without authorization of the owners.  The problem with example.edx is that it doesn't give any response, not even a 404, so it made it harder to test the responses.